nbcparser/pkg/parser/templates/cse_cmd.sh.gtpl

```diff
@@ -148,4 +148,7 @@ IS_KATA="{{.GetIsKata}}"
 ARTIFACT_STREAMING_ENABLED="{{.GetEnableArtifactStreaming}}"
 SYSCTL_CONTENT="{{getSysctlContent .CustomLinuxOsConfig.GetSysctlConfig}}"
 PRIVATE_EGRESS_PROXY_ADDRESS=""
+BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER=""
+ENABLE_IMDS_RESTRICTION="false"
```

Comment:
non-blocking: we should add some automation/PR-time checks to make sure this is in-sync with what we have in cse_cmd.sh

Reply:
Yes good point, will add this as a followup task